reaktive/src/commonMain/kotlin/com/badoo/reaktive/observable/ConcatMap.kt

```diff
@@ -32,7 +31,7 @@ private class ConcatMapObserver<in T, in R>(
 
     private val actor = serializer(::processEvent)
     private val innerObserver = InnerObserver(callbacks, actor).addTo(this)
-    private val queue = SharedQueue<T>()
```

Comment:
Remove `SharedQueue` implementation too?

Reply:
Oh, thanks. Missed that. Will do.

Reply:
Removed.